lib/writer.js

```diff
@@ -264,14 +260,14 @@ class ParquetTransformer extends stream.Transform {
 
   _transform(row, encoding, callback) {
     if (row) {
-      this.writer.appendRow(row).then(callback);
+      this.writer.appendRow(row).then(d => callback(null, d), callback)
```

Comment:
since `d` is always undefined, you can do:

```suggestion
      this.writer.appendRow(row).then(d => callback(), callback)
```